### PR TITLE
Qe 121 add storybook tests for the list radio question type

### PR DESCRIPTION
**Pull request by ramishenouda48**
1. Added the following tests for the optional list:
    1. expect the list radio question to have three answers
    2. expect the last list answer to have an empty value
2. Added the following test for the mandatory list:
     1. expect list radio question to have two answers
     2. expect the last list answer to have a value
3. Added the following basic tests:
     1. Expect input fields to be of type radio
     2. Expect input fields to have the same name
     3. Should be able to choose the first choice
     4. Should be able to choose the last choice after choosing an option
     5. Should be able to add new answers
     6. Should be able to choose the latest added option
     7. Should be able to delete the selected answer
 
list radio question tests

### CODE REVIEW

File: src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
 
 import { Language } from 'helpers'
-import { QuestionType } from 'components/Survey/QuestionTypes'
+import {
+  ListRadioQuestion,
+  QuestionType,
+} from 'components/Survey/QuestionTypes'
 
 import { Question } from '../../Questions/Question'
+import { within } from '@testing-library/react'
```
Comment (kevin-foster-uk) [inline src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js:10]: Import ordering. These are global and should be in the top section of imports.

File: src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js [hunk 2]
```diff
@@ -70,3 +95,165 @@ export const Basic = () => {
...
+      if (index !== 0) {
+        expect(inputField.checked).toBe(false)
+      }
+    })
+  })
+
+  await step(
+    'Should be able to choose the last choice after choosing an option',
+    () => {
+      const lastInptField = inputFields[inputFields.length - 1]
```
Comment (kevin-foster-uk) [inline src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js:184]: Search and replace "Inpt" to "Input"?

Comment (ramishenouda48) [reply src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js:184]: updated, thank you

File: src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js (hunk 1 above)
Comment (ramishenouda48) [reply src/components/Survey/QuestionTypes/ListRadioQuestion/ListRadioQuestion.stories.js:10]: updated, thank you

**Pull request merged**